packages/tools/kadena-cli/src/keys/utils/storage.ts

```diff
@@ -145,7 +145,7 @@ export async function readKeyFileContent(
   filePath: string,
 ): Promise<TSeedContent | IKeyPair | undefined> {
   const fileContents = await services.filesystem.readFile(filePath);
-  if (fileContents === null) {
+  if (!fileContents) {
```

Comment:
Is this the reason to have `strict-boolean-expressions` disabled?

This should be compared to `null` if that's what we expect since, `readFile: (path: string) => Promise<string | null>;` If you want to also compare for path length, you should use `if(fileContents === null && fileContents.length === 0)`.

You could consider `fs.exists` or something similar if you're actually interested in whether the file exists


Edit: I see you changed the signature of the function to return `void`. You can check for `undefined`

Reply:
nope, this is not the reason for disabling the rule. `!fileContents` works fine(without TS error) for both `undefined` or `null` case. 

i don't remember exactly why I disabled. I guess there were lots of warnings previously but not anymore.